src/Outsystems.SetupTools/Functions/New-OSPlatformPrivateKey.ps1

```diff
@@ -39,7 +39,8 @@ function New-OSPlatformPrivateKey
             return $null
         }
 
-        LogMessage -Function $($MyInvocation.Mycommand) -Phase 1 -Stream 0 -Message "Returning: $newKey"
+        $maskedNewKey = Mask-Key -Text $newKey
```

Comment:
Can you fix this?
Should be MaskKey right? Not Mask-Key.

Reply:
Yes, fixed!